.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+*Descriptive Summary of your Bugfix, Feature, or Refactoring.*
+
+## Checklist
+- [ ] Sensible git history (for example, squash "typo" or "fix" commits). See the
+      [Rewriting History](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) guide for
+      details.
+- [ ] update the changelog (if necessary)
```

Comment:
Minor nitpick: let's begin this sentence with a capital `U`, to be like the one above.